src/rdb.h

```diff
@@ -92,10 +92,12 @@
 #define RDB_TYPE_LIST_QUICKLIST 14
 #define RDB_TYPE_STREAM_LISTPACKS 15
 #define RDB_TYPE_HASH_LISTPACK 16
+#define RDB_TYPE_LIST_QUICKLIST2   17
```

Comment:
```suggestion
#define RDB_TYPE_LIST_QUICKLIST_2   17
```
Uniform format with `RDB_TYPE_ZSET_2`.